.github/workflows/build-releases.yml

```
@@ -0,0 +1,260 @@
+          
+  build-docker:
+    name: Build Docker Images
+    needs: build-matrix
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+    
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Download Linux artifacts
+        uses: actions/download-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)